zeebe/gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -224,11 +224,14 @@ message CreateProcessInstanceRequest {
   // will start at the start event. If non-empty the process instance will apply start
   // instructions after it has been created
   repeated ProcessInstanceCreationStartInstruction startInstructions = 5;
+
   // the tenant id of the process definition
   string tenantId = 6;
 
   // a reference key chosen by the user and will be part of all records resulted from this operation
   optional uint64 operationReference = 7;
+
+  repeated ProcessInstanceCreationRuntimeInstruction runtimeInstructions = 8;
```

Comment:
:x: We need to document what this is / what it can be used for (and we'll also need to document it [here](https://docs.camunda.io/docs/next/apis-tools/zeebe-api/gateway-service/))

Reply:
I'll add the in-place docs here, and I added a note for myself to add gRPC API docs as part of https://github.com/camunda/camunda-docs/issues/5798. I'd like to do it later, because the naming will change due to our recent decision to proceed with termination.